src/serializers/computed_fields.rs

```diff
@@ -73,6 +73,10 @@ impl ComputedFields {
         }
         for computed_field in &self.0 {
             let property_name_py = computed_field.property_name_py.as_ref(model.py());
+            let value = model.getattr(property_name_py).map_err(py_err_se_err)?;
+            if extra.exclude_none && value.is_none() {
+                return Ok(());
```

Comment:
As per https://github.com/pydantic/pydantic/issues/8691, should this be a `continue` instead of `return Ok(());`?